cobc/ChangeLog

```diff
@@ -1,3 +1,10 @@
+2025-11-10  Emilien Lemaire <emilien.lemaire@ocamlpro.com>
```

Comment:
one empty line before and after the Changelog entry, please (currently it is 0/2)